### DISCUSSION TITLE
Why dropdown is not showing after I login?

### DISCUSSION THREAD

**Discussion by krxshia** [twbs/bootstrap#38915] [Q&A]
So I created a code where after the user logged in, the dropdowns will show instead of redirecting them to login page.

Here's my code:

<img width="514" alt="image" src="https://github.com/twbs/bootstrap/assets/90703249/593743e1-e688-4433-8a61-e54827a922a2">

I already have session_start(); on my file.

**Comment by patkon** [comment]
Have you checked the value of your variables? `$current_page` and `$_SESSION['user_id']`

I can not evaluate it now from the screenshot but did you wrap this dropdown menu in a `.dropdown` element?
From the Docs:

> Wrap the dropdown’s toggle (your button or link) and the dropdown menu within .dropdown, or another element that declares position: relative;. Ideally, you should use a `<button>` element as the dropdown trigger, but the plugin will work with `<a>` elements as well. The examples shown here use semantic `<ul>` elements where appropriate, but custom markup is supported.
